src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+form {
+  position: relative;
+  margin: 20px 0;
+}
+
+input {
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: none;
+  outline: 0;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.big_container::before {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 25.5px;
+  left: 26px;
+  z-index: 1;
```

Comment:
do you really need this?

Reply:
yes https://prnt.sc/takjf3

Reply:
I changed ::before on <image> element. So that i need not z-index anymore, but I need position: absolute.
Without this image will not cover part of input.

And with position relative there will remain space of image in its started position. I think absolute position is the best way